src/Robo/Plugin/Commands/DevelopmentModeBaseCommands.php

```diff
@@ -253,9 +253,9 @@ protected function landoUri($siteDir): string
                 $domain = array_pop($siteDomains);
                 return "http://$domain";
             }
-        } elseif ($uri = $landoCfg['services']['appserver']['overrides']['environment']['DRUSH_OPTIONS_URI'] ?? null) {
+        } elseif (isset($landoCfg['services']['appserver']['overrides']['environment']['DRUSH_OPTIONS_URI'])) {
```

Comment:
Just confirming that we wanted to change the logic here. As we returned `null` before without getting to the `else` below. 

I think this is better though FWIW.

Reply:
> As we returned null before without getting to the else below.

That is not my reading of it. If it was not set, the null coalesce would have returned NULL, but then the `elseif` would have failed the check and it would have moved on to the `else`.

Reply:
👍🏼